ports/net/macathena-locker-support/Portfile

```diff
@@ -3,7 +3,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        macathena locker-support 8bebaf104fea5d5bbeed9eb6ff57e902cd79d2e9
+github.setup        macathena locker-support 3801013b44488dab81bcea2b72634ba9340a1499 
 
 name                macathena-locker-support
 version             10.4.8
```

Comment:
If you're changing the source, you need to either raise the version or the revision. (In this case, probably revision is the appropriate one since there hasn't been a new upstream release.)